pkg/cloud/cloud.go

```diff
@@ -167,6 +168,10 @@ var (
 
 	// VolumeNotBeingModified is returned if volume being described is not being modified
 	VolumeNotBeingModified = fmt.Errorf("volume is not being modified")
+
+	// ErrSnapshotRateLimit is returned when cloud.go refuses to send a CreateSnapshot AWS API request
+	// Used to work around https://github.com/kubernetes-csi/external-snapshotter/issues/778
+	ErrSnapshotRateLimit = fmt.Errorf("Refusing to send CreateSnapshot to AWS (see https://github.com/kubernetes-sigs/aws-ebs-csi-driver/issues/1608#issuecomment-1554748900)")
```

Comment:
Error messages should not contain external links; they should be descriptive enough on their own. Suggest changing this to something like

```
fmt.Errorf("snapshot creation rate-limited")
```